### PR TITLE
Circuit logging slight expansion

### PR DESCRIPTION
**Pull request by aidan-bowman**
## About The Pull Request

When plopping a battery into an assembly, it logs the player, the battery, and the assembly into the GAME log file.

## Why It's Good For The Game

better logging. since batteries are what "activate" assemblies, the person who put the battery in is the person who actually activated a circuit, not the guy who printed the assembly.

## Changelog
:cl:
admin: circuit's log when a battery is put in
/:cl:

### CODE REVIEW

**Comment by aidan-bowman** [comment]
you're such a sussy baka

**Review by ike709 — CHANGES_REQUESTED** [review]
Don't bloat the game log, use `investigate_log()` with `INVESTIGATE_CIRCUIT`.

**Comment by aidan-bowman** [comment]
ah alright i'll do that soon

**Comment by aidan-bowman** [comment]
fixed

File: code/modules/integrated_electronics/core/assemblies.dm [hunk 1]
```diff
@@ -669,6 +669,7 @@
 		diag_hud_set_circuitstat() //update diagnostic hud
 		playsound(get_turf(src), 'sound/items/Deconstruct.ogg', 50, 1)
 		to_chat(user, "<span class='notice'>You slot the [I] inside \the [src]'s power supplier.</span>")
+		investigate_log("[user] has slotted [I] into [src].", INVESTIGATE_CIRCUIT)
```
Comment (ike709) [inline code/modules/integrated_electronics/core/assemblies.dm]:
```suggestion
		investigate_log("[key_name(user)] has slotted [I] into [src].", INVESTIGATE_CIRCUIT)
```

**Review by ike709 — APPROVED** [review]

**Comment by aidan-bowman** [comment]
so uh, now that it's approved, is it gonna get merged?

**Pull request merged**